### PR TITLE
Make socket.on/socket.addListener chainable. Fixes #81

### LINKED ISSUE #81: Make socket.on/socket.addListener chainable
It would be nice to make socket.on/socket.addListener chainable like EventEmitter, so we can write:

```
socket
  .on('event1', handler1)
  .on('event2', handler2)
  ...
```

### PR DESCRIPTION
**Pull request by jacek-pulit**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+.idea
```
Comment (btford) [inline .gitignore]: these changes are unrelated

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-socket-io",
-  "version": "0.7.0",
+  "version": "0.7.1",
```
Comment (btford) [inline bower.json]: you should not bump versions in a PR

**Comment by btford** [comment]
also needs a test

**Comment by pciazynski** [comment]
Thank you! This is very important! I think that this should be merged. I spend over 3 hours in searching bugs in my code, and the problem was, that angular-socket.io on event wasn't chainable ;P

**Comment by jacek-pulit** [comment]
Actally I have created another pull request including tests here https://github.com/btford/angular-socket-io/pull/124
This one is closed.
